### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by kinafu**
looked for some (hopefully) better-fitting synonyms of English words
setup instructions: added how to enter backup details into  config file
some minor contentual addition for faster comprehension

### CODE REVIEW

**Comment by kinafu** [comment]
All these commits do not matter, just the readme changed :)
See commit 8fce9336a83bd0d70f21fcaf7483c4b1a37d7168

File: README.md [hunk 1]
```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
...
 
 1. For logging: Create `/var/log/borg` with appropiate permissions.
 2. Create a subdirectry called `/var/log/borg/last` (configurable as [`LAST_BACKUP_DIR`](borgcron.sh#L9)). There the `.time` files will be written to containing the tiome of the last backup execution.
-3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar depending on your shell). It will use the `.time` files to check the latest execution of the backups. You may also configure it's [`CRITICAL_TIME`](tools/checklastbackup.sh#L8), which defines the time the backup notifies you of a problem.
+3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar, depending on your shell). It will read the `.time` files to display the time of  the last execution of your backups, when you login into your shell. You may also set a period of time ([`CRITICAL_TIME`](tools/checklastbackup.sh#L8)) in order to get a notification, if no successful backup has been made within that time.
```
Comment (rugk) [inline README.md:82]: Please make clear the "period of time" (maybe [time period](https://dict.leo.org/englisch-deutsch/time+period) is better) is connected/inside of the login script you mention in the previous sentence. Currently it may look like this period is not related to the script.
Simply change "set a period" to "set the period" e.g.

File: README.md [hunk 2]
```diff
@@ -7,12 +7,12 @@ They add some convienent features around borg, regarding environments with only
...
-* **[Automatic retries](#less-maintenance-more-safety):** When backups [stop mid-way](https://borgbackup.readthedocs.io/en/stable/faq.html#if-a-backup-stops-mid-way-does-the-already-backed-up-data-stay-there) they are automatically restarted.
-* **[Easy configuration](config/example-backup.sh):** Using shell-scripts you can configure each backup and then execute it the order/way you want.
+* **[Automated retries](#less-maintenance-more-safety):** When backups [stop mid-way](https://borgbackup.readthedocs.io/en/stable/faq.html#if-a-backup-stops-mid-way-does-the-already-backed-up-data-stay-there) they are automatically restarted.
+* **[Simple configuration](config/example-backup.sh):** Using shell-scripts you can configure each backup and then execute it the order/way you want.
 * **[Status information](#less-maintenance-more-safety):** You can use a login script to get a notice when backups failed.
 * **[Optional & adjustable](#modular-approach):** You do not have to use all features and you can adjust them in a simply way.
 
-### Local lock
+### Local lock (only for borg version < 1.1.0)
```
Comment (rugk) [inline README.md:15]: shorter title: "borg version < 1.1.0"

File: README.md [hunk 3]
```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
...
 
 1. For logging: Create `/var/log/borg` with appropiate permissions.
 2. Create a subdirectry called `/var/log/borg/last` (configurable as [`LAST_BACKUP_DIR`](borgcron.sh#L9)). There the `.time` files will be written to containing the tiome of the last backup execution.
-3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar depending on your shell). It will use the `.time` files to check the latest execution of the backups. You may also configure it's [`CRITICAL_TIME`](tools/checklastbackup.sh#L8), which defines the time the backup notifies you of a problem.
+3. Include/add the [`tools/checklastbackup.sh`](tools/checklastbackup.sh) script to your `~/.bashrc`, `~/.zshrc` or similar, depending on your shell). It will read the `.time` files to display the time of  the last execution of your backups, when you login into your shell. You may also set a period of time ([`CRITICAL_TIME`](tools/checklastbackup.sh#L8)) in order to get a notification, if no successful backup has been made within that time.
 
-### 3. Setup local log (optional)
+### 4. Setup local log (recommended for borg version **<** 1.1.0)
```
Comment (rugk) [inline README.md:84]: You already mentioned this previously, so do not mention it again in the headline.
Also it is still "optional", so keep this label there.

File: README.md [hunk 4]
```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
...
 
-### 4. Setup MySQL dump (optional)
+### 5. Setup MySQL dump (optional)
 
-The [`databasedump.sh`](tools/databasedump.sh) script can be used to dump the database into a dir before executing the backup. You can do this either directly before running the backup by including the file in your config file, or you can setup a new cron job using another user, who is allowed to dump the databases into a dir. The cron job should, of course, be executed before executing the actual backup.
+The [`databasedump.sh`](tools/databasedump.sh) script can be used to dump your database(s) into a dir before executing the backup. You can do this either directly before running the backup by including the content of [`databasedump.sh`](tools/databasedump.sh) in your config file, or you can setup a new cron job using another user, who is allowed to dump the databases into a dir. The cron job should, of course, be executed before executing the actual backup.
```
Comment (rugk) [inline README.md:97]: No you absolutely do not have to include the content of that file. Just include (aka "source it" – using the `.` or `source` command).

File: README.md [hunk 5]
```diff
@@ -69,32 +69,36 @@ Also, you can of course not use some features outlined here. That's why the whol
...
 
-### 5. Setup cron/anacron
+### 6. Setup cron/anacron
 
-Finally test the backup process and add the scripts to cron (use the `crontab -e` command to edit the files): 
+Finally test the backup process,before you add the scripts into cron (use the `crontab -e` command to edit the files):
```
Comment (rugk) [inline README.md:101]: `,<space>`

"into cron" sounds strange anyway. Maybe ["setup cron"](https://askubuntu.com/questions/2368/how-do-i-set-up-a-cron-job) or "add an entry [**to** my cron](http://unix.stackexchange.com/questions/21297/ddg#21300)…

**Review by rugk — CHANGES_REQUESTED** [review]

**Comment by rugk** [comment]
I've adjusted the things by myself now, as the setup likely moves into the wiki anyway. (see https://github.com/rugk/borg-cron-helper/issues/13). There editing it will be faster…
